static/js/components/UserListCard.js

```diff
@@ -28,19 +32,62 @@ type Props = {
   userList: UserList
 }
 
+function ConfirmDeleteDialog(props) {
+  const { userList, close } = props
+
+  const [, deleteUserList] = useMutation(deleteUserListMutation)
+
+  return (
+    <Dialog
+      title={`Delete this ${readableLearningResources[userList.object_type]}?`}
```

Comment:
Use `list_type` instead of `object_type`

Reply:
:grimacing: